spec/lucky/select_helpers_spec.cr

```diff
@@ -5,6 +5,21 @@ private class TestPage
 
   def render
   end
+
+  def select(field)
+    select_input field do
+    end
+    self
+  end
+
+  def options(field, options)
```

Comment:
And `render_options`? That way it's clear that it isn't using the `options` and `select` tags